tests/mocks.py

```diff
@@ -0,0 +1,98 @@
+from typing import Dict, List, Set
```

Comment:
These by convention go into `tests/support` (and probably separate modules, one for Kubernetes and one for Docker).